src/app/learn/get-started/advanced/accessibility-unit-tests/index.html

```diff
@@ -0,0 +1,145 @@
+<stache
+  pageTitle="Accessibility Tests"
+  navTitle="Accessibilty tests"
+  showTableOfContents="true"
+  navOrder="1">
+  
+  <stache-page-summary>
+    <p>
+      The <a href="https://github.com/blackbaud/skyux-lib-testing">SKY UX testing library</a> provides methods to run <a href="https://github.com/dequelabs/axe-core">aXe automated accessibility tests</a> in your unit tests. aXe is an open source library of automated accessibility rules that is integrated into SKY UX to identify potential issues for a subset of the <a href="https://www.w3.org/TR/WCAG20/">Web Content Accessibility Guidelines 2.0 Level A and AA success criteria</a>. For SKY UX guidance on designing, coding, and testing accessible experiences, see the accessibility section.
+    </p>
+  </stache-page-summary>
+
+  <p>
+    <stache-include fileName="e2e-recommendation.html"></stache-include> If you must use e2e tests, the deprecated accessibilty analyzer lets you add accessibility checks to e2e specs and manage rules for accessibility tests with an <stache-code>a11y</stache-code> configuration property. For more information, see <a routerLink="/learn/get-started/advanced/accessibility-analyzer">the accessibility analyzer</a> and <a routerLink="/learn/reference/configuration">the configuration guidelines</a>.
```

Comment:
- "deprecated accessibilty analyzer" Misspelled

Reply:
Corrected the spelling of "accessibility."